internal/db/postgres/transformers/masking.go

```diff
@@ -34,7 +34,7 @@ const (
 	MMobile     string = "mobile"
 	MTelephone  string = "tel"
 	MID         string = "id"
-	MCreditCard string = "credit_cart"
```

Comment:
There is a typo as well in the parameter description

https://github.com/GreenmaskIO/greenmask/blob/9e4afa1ac854f2fad42c905e605e3b976381c677/internal/db/postgres/transformers/masking.go#L60